README.md

```diff
@@ -413,27 +431,29 @@ class Profile::Create < Opera::Operation::Base
   end
 
   def build_record
-    context[:profile] = dependencies[:current_account].profiles.build(params)
-    context[:profile].force_name_validation = true
+    self.profile = current_account.profiles.build(params)
+    self.profile.force_name_validation = true
   end
 
   def create
-    context[:profile] = context[:profile].save
+    self.profile = profile.save
     finish
```

Comment:
if `finished` is removed let's remove it from examples.

Reply:
Done: https://github.com/Profinda/opera/pull/5/files#diff-51d3653f611ad330e64b154eab9ab5d59fa90bfe203af867b9a754a1f7c96c97L28